.gitlab-ci.yml

```diff
@@ -1,37 +1,591 @@
+  services:
+  - docker:dind
+  tags:
+    - mender-qa-slave
+  before_script:
+    - export WORKSPACE=${CI_PROJECT_DIR}/..
+    - export GOPATH="$WORKSPACE/go"
+    # This template is used in both build and acc test stages
+    - test -n "$ONLY_BUILD" && export TEST_QEMUX86_64_UEFI_GRUB=false
+    - test -n "$ONLY_BUILD" && export TEST_VEXPRESS_QEMU=false
+    - export RUN_INTEGRATION_TESTS=false
+    # Prepare tzdata unatended configuration
+    - ln -fs /usr/share/zoneinfo/Europe/Oslo /etc/localtime
+    # basic tools
+    - apt-get update -q
+    - apt-get install -qqy
+      git wget gnupg2 pass autoconf automake build-essential diffstat gawk chrpath
+      libsdl1.2-dev e2tools nfs-client s3cmd psmisc screen libssl-dev python-dev
+      libxml2-dev libxslt-dev libffi-dev nodejs libyaml-dev sysbench texinfo pkg-config
+      zlib1g-dev libaio-dev libbluetooth-dev libbrlapi-dev libbz2-dev libglib2.0-dev
+      libfdt-dev libpixman-1-dev zlib1g-dev jq liblzo2-dev device-tree-compiler
+      qemu-system-x86 bc kpartx liblzma-dev cpio sudo
+    # Prepare mender user
+    - useradd -m -u 1010 mender || true
+    - mkdir -p /home/mender/.ssh
+    - echo "mender ALL=(ALL:ALL) NOPASSWD:ALL" >> /etc/sudoers
+    - sed -i -e 's/^\( *Defaults *requiretty *\)$/# \1/' /etc/sudoers
+    - chown -R mender:mender /home/mender
+    - chown -R mender:mender ${WORKSPACE}
+    - usermod -a -G docker mender
+    # docker-compose from upstream
+    - curl -L "https://github.com/docker/compose/releases/download/1.24.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    - chmod +x /usr/local/bin/docker-compose
+    # golang from upstream
+    - wget -q https://storage.googleapis.com/golang/go1.12.linux-amd64.tar.gz
+    - gunzip -c go1.12.linux-amd64.tar.gz | (cd /usr/local && tar x)
+    - ln -sf /usr/local/go/bin/go /usr/local/bin/go
+    - ln -sf /usr/local/go/bin/godoc /usr/local/bin/godoc
+    - ln -sf /usr/local/go/bin/gofmt /usr/local/bin/gofmt
+    # Python 2 pip
+    - apt-get install -qqy python-pip
+    - pip install --upgrade pip
+    - pip2 install requests --upgrade
+    - pip2 install pytest --upgrade
+    - pip2 install filelock --upgrade
+    - pip2 install pytest-xdist --upgrade
+    - pip2 install pytest-html --upgrade
+    - pip2 install -I fabric==1.14.0
+    - pip2 install psutil --upgrade
+    - pip2 install boto3 --upgrade
+    - pip2 install pycrypto --upgrade
+    # Python 3 pip
+    - apt-get install -qqy python3-pip
+    - pip3 install --upgrade pyyaml
+    # Locales
+    - apt-get install locales
+    - locale-gen --purge en_US.UTF-8
+    - export LC_ALL=en_US.UTF-8
+    - export LANG=en_US.UTF-8
+    - export LANGUAGE=en_US.UTF-8
+    # debugfs
+    - cp /sbin/debugfs /usr/bin/ || echo "debugfs not in /sbin/"
+    # sysstat monitoring suite
+    # collect cpu, load avg, memory and io usage every 2 secs forever
+    # use 'sadf' from sysstat to render the result file (~/sysstat.log) manually
```

Comment:
`sadf`? Or `sar`?

Reply:
TBH, I have no idea. That is copied as is in the `jenkins-yoctobuild-init-script.sh`

Reply:
Should be `sar` :+1: I am fixing it in both palces